src/main/java/frc/utils/ctre/BusChain.java

```diff
@@ -7,56 +7,70 @@
 
 public enum BusChain {
 
-	ROBORIO("rio"),
+	ROBORIO("Rio"),
```

Comment:
```suggestion
	ROBORIO("rio"),
```
the chain name if how the can call itself or ctre call it, its not our choice *it ccan be but rename chain is something i dont have power to do)

Reply:
hillel told me to change it idk fixed
also come back to dis teach me how to score a trap